### ISSUE TITLE
Remove the print('')

### ISSUE THREAD

**Issue by KingDoxik** [jamesblasco/snap_scroll_physics#1]
Thanks for this library. Would it be possible to remove the print statement?

**Comment by raulmabe** [comment]
Same petition here! The library is incredibly useful, but the prints are bloating my output 

```
1587 (number of prints) flutter: p 420.0, 253.2
1791 (number of prints) flutter: p 420.0, 253.2
```

**Comment by jamesblasco** [comment]
Sorry about that! Fixed on `0.0.1+3`
